jui-core/index.d.ts

```diff
@@ -33,9 +33,9 @@ interface UtilBase {
      * @param {Function} ctor base Class
      * @param {Function} superCtor super Class
      */
-    // tslint:disable-next-line:callable-types
+    /* tslint:disable-next-line:callable-types */
```

Comment:
I runned  `npm run list jui-core` in local terminal. 

But there is no error message. 

If you have documentation related to this, please show the link.